src/components/Layout/Navigation/RecentList.tsx

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2018-2020 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+
+import React from 'react';
+import {
+  NavGroup,
+  NavList,
+} from '@patternfly/react-core';
+import { PlusIcon } from '@patternfly/react-icons';
+
+import NavigationRecentItem from './RecentItem';
+import { NavigationItemObject, NavigationRecentItemObject } from '.';
+import NavigationMainItem from './MainItem';
+
+import styles from './index.module.css';
+
+function buildCreateWorkspaceItem(): React.ReactElement {
+  const item: NavigationItemObject = {
+    to: '/get-started#custom-workspace',
```

Comment:
It should be getting from ```enum ROUTE```
https://github.com/che-incubator/che-dashboard-next/pull/48/files#diff-510ca6cdc30d8b32c0054feb11b8e8f2R15

Reply:
WDYT?

Reply:
good point, thanks